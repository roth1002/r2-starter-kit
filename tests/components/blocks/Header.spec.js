import React from 'react';
import { shallow } from 'enzyme';
import Component from 'components/blocks/Header';

describe('<Header />', () => {
  const initProps = {
  };

  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<Component {...initProps} />);
  });

  it('should render initial component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
