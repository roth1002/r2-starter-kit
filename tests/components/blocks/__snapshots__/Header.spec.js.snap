// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render initial component 1`] = `
<div
  className="default"
>
  <a
    className="github"
    href="https://github.com/tpai/r2-starter-kit"
    rel="noopener noreferrer"
    target="_blank"
  >
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "test-file-stub",
        }
      }
    />
  </a>
  <a
    href="https://jsonplaceholder.typicode.com/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      className="api"
      src="test-file-stub"
    />
  </a>
</div>
`;
